json_schema_fuzz/utils.py

```diff
@@ -0,0 +1,28 @@
+""" Utility functions """
+
+
+def merge(a, b, path=None, update=True):
```

Comment:
I like the idea here. I think this may be the right way to solve `allOf`, as well as some other schema-combination issues. And I think this does 90% of what we want it to do. But I'm afraid that last 10% is going to be very tricky. For example, `"allOf": [{"multipleOf": 2}, {"multipleOf": 3}]`. There are a couple projects out there that claim to do this, so we could look around for inspiration.